app/jobs/cron_job.rb

```diff
@@ -40,16 +40,13 @@ def perform
   end
 
   class ReminderJob < CronJob
-    # Every hour at minute 57 (00:57:, 01:57, 02:57 [...] 23:57)
-    self.cron_expression = "57 * * * *"
+    # At 3:00 every day
+    self.cron_expression = "0 3 * * *"
 
     def perform
-      rdvs_starting_in_48_to_49_hours = Rdv.where("starts_at >= ? AND starts_at < ?", 48.hours.from_now, 49.hours.from_now)
-      rdvs_starting_in_48_to_49_hours.not_cancelled.each do |rdv|
-        Notifiers::RdvUpcomingReminder.perform_with(rdv, nil)
-      rescue StandardError => e # don't interrupt the whole loop when one RDV crashes
-        Sentry.add_breadcrumb(Sentry::Breadcrumb.new(message: "Could not send reminder", data: { rdv_id: rdv.id }))
-        Sentry.capture_exception(e)
```

Comment:
Ce `rescue` avait été ajouté pour éviter que le job ne crashe si l'une des itérations de la boucle crashe. Cela pouvait arriver car chaque itération de la boucle appelait `Notifiers::RdvUpcomingReminder.perform_with` (qui fait plein de choses) de façon synchrone.

Puisque désormais la saule chose faite est ajoutée un job à la file, il n'est pas nécessaire de s'inquiéter qu'un exception soit levée. J'ai donc supprimé ce `rescue` et supprimé la section de test nommée `"error handling"`.